pkg/threescale/limits_storage_in_memory.go

```diff
@@ -0,0 +1,36 @@
+package threescale
+
+import (
+	"time"
+
+	gocache "github.com/patrickmn/go-cache"
+)
+
+type inMemoryLimitsStorage struct {
+	internalStorage *gocache.Cache
+}
+
+func newInMemoryLimitsStorage() *inMemoryLimitsStorage {
+	goCache := gocache.New(gocache.NoExpiration, time.Minute)
+	return &inMemoryLimitsStorage{internalStorage: goCache}
+}
+
+func (storage *inMemoryLimitsStorage) get(key string) (int, bool, error) {
+	val, exists := storage.internalStorage.Get(key)
```

Comment:
Not important, but I think this does look cleaner: 

```golang
if val, ok := storage.internalStorage.Get(key); !ok { 
   return 0, false, nil
}
```

Reply:
I don't have a strong preference. I think I tend to write it in the other form because it's more similar as how it would be in the majority of programming languages.

Instead of changing this, I think we should adopt a linter to have a consistent style. What do you think?